client/state/data-layer/wpcom/read/teams/test/index.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import useNock from 'test/helpers/use-nock';
+
+import { handleTeamsRequest } from '../';
+import { requestTeams } from 'state/reader/teams/actions';
+import { READER_TEAMS_RECEIVE } from 'state/action-types';
+
+export const successfulResponse = {
+	teams: [
+		{
+			title: 'Automattic',
+			slug: 'a8c'
+		}
+	],
+	number: 1
+};
+
+describe( 'wpcom-api', () => {
+	const nextSpy = sinon.spy();
+
+	describe( 'teams request', () => {
+		useNock( nock => (
+			nock( 'https://public-api.wordpress.com:443' )
+				.get( '/rest/v1.2/read/teams' )
+				.reply( 200, successfulResponse )
+				.get( '/rest/v1.2/read/teams' )
+				.reply( 500, new Error() )
+		) );
+
+		it( 'should dispatch RECEIVE action when request completes', ( done ) => {
+			const dispatch = sinon.spy( action => {
+				if ( action.type === READER_TEAMS_RECEIVE ) {
+					expect( dispatch ).to.have.been.calledWith( {
+						type: READER_TEAMS_RECEIVE,
+						payload: successfulResponse,
+					} );
```

Comment:
yup. if you don't, other things down the chain won't see the action. Should always be tested.